src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+import Duke.task.Deadlines;
+import Duke.task.EmptyTaskException;
+import Duke.task.Events;
+import Duke.task.Task;
+import Duke.task.ToDos;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public static ArrayList<Task> taskList;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greeting = "    ____________________________________________________________\n" +
+                "     Hello! I'm Duke\n" +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________\n";
+        String bye = "     Bye. Hope to see you again soon!\n" +
+                "    ____________________________________________________________";
+        String line = "    ____________________________________________________________\n";
+        String command = "null";
+        String spaces = "    ";
+
+        taskList = new ArrayList<>();
+
+        Scanner in = new Scanner(System.in);
+
+        FileClass file = new FileClass("data/tasks.txt");
+
+        System.out.println(greeting);
+        while (!command.equals("bye")) {
+            command = in.nextLine();
+            if (command.equals("bye")) {
+                System.out.println(line + bye);
+            } else if(command.equals("list")){
```

Comment:
Maybe you should consider improve the level of abstractions. You method seems quite huge. Please consider create methods for each command type e.g printList(parameters), deleteTask(parameters) and etc.